apps/storefront/pages/bloggen/051223-miro-workshop.mdx

```diff
@@ -0,0 +1,78 @@
+import { BlogArticleLayout } from '@layouts';
+import { Meta, ResponsiveIframe, Image } from '@components';
+import { Contributors } from '@blog';
+
+<Meta
+  title='Derfor trenger vi et felles designsystem'
+  description='Høsten 2023 arrangerte vi en åpen presentasjon og mini-workshop om felles designsystem. Over 200 deltok og vi fikk 440 tilbakemeldinger på gevinster og utfordringer.'
+/>
```

Comment:
Maybe we should have a meta image for this?
If you do `image='/img/bloggen/miro.png'`, the miro image will be added.

Reply:
Thanks, I fix :)